packages/menu/src/Menu/Menu.tsx

```diff
@@ -286,12 +287,12 @@ export function Menu({
       case keyMap.Tab:
         e.preventDefault(); // Prevent tabbing outside of portal and outside of the DOM when `usePortal={true}`
         handleClose();
-        setFocus((refEl || triggerRef)?.current); // Focus the trigger on close
+        setFocus((triggerRef || refEl)?.current); // Focus the trigger on close
```

Comment:
why are we changing the order here?

Reply:
Since i'm using both a `triggerRef` and `refEl` inside `SplitButton`, the focus was going to the refEl on close. This way, the focus is on the trigger.

Reply:
i think this could be a breaking change? is there a way to use the refEl to point to the container, not the buttons themselves?

Reply:
Removing this